content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,49 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The control plane can [bind PVCs to matching PVs](#binding) in the
+cluster. However, if you want a PVC to bind to a specific PV, you need to
+pre-bind them. As it's a bi-directional binding between the PVC and the PV,
+there are two ways:
+
+- Specify the PV in your PVC using the `volumeName` field
```

Comment:
I believe this first method doesn't guarantee the PV right? You could have:
* 2 pvcs pointing to the same pv
* a second pvc that doesn't specify any pv, but matches it.

Should we just recommend the 2nd way of setting claimRef?

Reply:
yes, the first method doesn't guarantee.

How about documenting it but we emphasize that it doesn't guarantee the PV right?

Sometimes it's useful as it is simpler and does not require `UPDATE` permission on cluster resource `PV`.

Reply:
maybe it's better not to say there are two ways.
I made an update. https://github.com/kubernetes/website/pull/22946/commits/839184d4098a648ac5ff14c3b13a8ebc6bc351ea
what do you think about it?